src/main/groovy/org/mini2Dx/butler/task/PushTask.groovy

```diff
@@ -49,11 +49,11 @@ class PushTask extends DefaultTask  {
 	}
 	
 	def pushAnyOsBuild() {
-		if(project.getExtensions().findByName('butler').anyOs == null) {
+		if(project.getExtensions().findByName('butler').anyOs.binDirectory == null) {
```

Comment:
This line shouldn't be changed. Or, it should be included after the original line. The original line allows anyOs to be optionally defined.